ReactNativeClient/lib/services/KeymapService.ts

```diff
@@ -75,6 +79,20 @@ const defaultKeymap = {
 	],
 };
 
+export class KeymapError extends Error {
+	// Translated error message for the GUI
+	private altMessage_: string;
+
+	constructor(message: string, altMessage: string) {
+		super(message);
+		this.altMessage_ = altMessage;
+	}
+
+	get altMessage() {
+		return this.altMessage_;
+	}
+}
+
```

Comment:
Please remove the altMessage property - error should only have one message. In fact, as far as I can see there's no actual use for the KeymapError object anymore so please remove KeymapError and use a regular Error object.

To throw the error, please use this:

```js
new Error(_('Accelerator "%s" is already used for command "%s".', item.accelerator, originalItem.command));
```

Reply:
Thanks for mentioning the example. I agree that it's unneccessary. My initial idea was give a friendly error message in the GUI. 
I'll replace them with one message.